tests/test_operators.py

```diff
@@ -45,23 +46,86 @@ def compare_streams(self, all, expected_all):
             self.assertDictEqual(input_dict, output_dict)
 
     def test_map_instance_values(self):
+        mappers = {"a": {"1": "hi", "2": "bye"}}
+
         inputs = [
-            {"a": 1, "b": 2},
-            {"a": 2, "b": 3},
+            {"a": "1", "b": "2"},
+            {"a": "2", "b": "3"},
         ]
 
         targets = [
-            {"a": "hi", "b": 2},
-            {"a": "bye", "b": 3},
+            {"a": "hi", "b": "2"},
+            {"a": "bye", "b": "3"},
         ]
 
+        # simple value substitute
         check_operator(
-            operator=MapInstanceValues(mappers={"a": {"1": "hi", "2": "bye"}}),
+            operator=MapInstanceValues(mappers=mappers),
             inputs=inputs,
             targets=targets,
             tester=self,
         )
 
+        # process_every_value=True would not accept non-list inputs
+        with self.assertRaises(ValueError):
+            check_operator(
+                operator=MapInstanceValues(mappers=mappers, process_every_value=True),
+                inputs=inputs,
+                targets=targets,
+                tester=self,
+            )
+
+        # strict is True by default, input value "3" in field "a" is missing from the mapper of "a"
+        with self.assertRaises(KeyError) as ke:
+            operator = MapInstanceValues(mappers=mappers)
+            operator.process(instance={"a": "3", "b": "4"})
+
+        inputs_p_e_v = [
```

Comment:
Why name is p_e_v?

Reply:
to be used with example case with process_every_value  is True.  Will extend to full name..